test/test_core.py

```diff
@@ -0,0 +1,16 @@
+from pyconnect.core import hash_sensitive_values
+
+
+def test_hash_sensitive_values_hashes():
+    config = {"sasl.password": "unhashed password", "regular_key": "regular value"}
+    config_copy = config.copy()
```

Comment:
```suggestion
```
Just use config... it looks a little bit confusing this way

Reply:
👍